site/quickstart.md

```diff
@@ -138,9 +138,13 @@ a.	Remove the domain's Ingress by using `helm`:
 ```
 helm delete --purge domain1-ingress
 ```
-b.	Remove the domain resource by using the sample [`delete-weblogic-domain-resources`](../kubernetes/samples/scripts/delete-domain/delete-weblogic-domain-resources.sh) script.
+b.	Remove the domain resources by using the sample [`delete-weblogic-domain-resources`](../kubernetes/samples/scripts/delete-domain/delete-weblogic-domain-resources.sh) script.
 
 c.	Use `kubectl` to confirm that the server pods and domain resource are gone.
+```
+kubectl get pods -n sample-domains-ns1
```

Comment:
helm and kubectl commands should have a '$' in front.

Reply:
Got it.